package pl.on.full.hack.auth.exception;

public class UserAlreadyExistsException extends RuntimeException {
}
